### ISSUE TITLE
Trollager particles don't like Air

### ISSUE THREAD

**Issue by naqaden** [Daveyx0/PrimitiveMobs#124]
[EntityTrollager #L449](https://github.com/Daveyx0/PrimitiveMobs/blob/85ad4e1af7cd4050421677af495d2db26740430f/src/main/java/net/daveyx0/primitivemobs/entity/monster/EntityTrollager.java#L449) in some situations determines the thrown block to be Air, making the resulting BLOCK_CRACK particle unhappy. It's been hard for me to reproduce though.

![screenshot](https://i.imgur.com/LMOh2Zv.png)

```
java.util.concurrent.ExecutionException: net.minecraft.util.ReportedException: Exception while adding particle
    at java.util.concurrent.FutureTask.report(Unknown Source)
    at java.util.concurrent.FutureTask.get(Unknown Source)
    at net.minecraft.util.Util.runTask(SourceFile:531)
    at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1087)
    at net.minecraft.client.Minecraft.run(Minecraft.java:3940)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
    at org.multimc.EntryPoint.listen(EntryPoint.java:143)
    at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: net.minecraft.util.ReportedException: Exception while adding particle
    at net.minecraft.client.renderer.RenderGlobal.spawnParticle(RenderGlobal.java:3003)
    at net.minecraft.client.renderer.RenderGlobal.spawnParticle(RenderGlobal.java:2976)
    at net.minecraft.world.World.spawnParticle(World.java:1169)
    at net.minecraft.world.World.spawnParticle(World.java:1148)
>   at net.daveyx0.primitivemobs.message.MessagePrimitiveParticle$Handler.lambda$onMessage$0(MessagePrimitiveParticle.java:94)
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    at java.util.concurrent.FutureTask.run(Unknown Source)
    at net.minecraft.util.Util.runTask(SourceFile:529)
    ... 17 more
Caused by: java.lang.ArrayIndexOutOfBoundsException: 0
    at net.minecraft.client.particle.ParticleDigging$Factory.createParticle(SourceFile:105)
    at net.minecraft.client.particle.ParticleManager.spawnEffectParticle(ParticleManager.java:151)
    at net.minecraft.client.renderer.RenderGlobal.spawnParticle0(RenderGlobal.java:3098)
    at net.minecraft.client.renderer.RenderGlobal.spawnParticle(RenderGlobal.java:2983)
    ... 24 more
```

**Comment by MasterEnderman** [comment]
Crash Report: https://paste.dimdev.org/jazamevoga.mccrash

**Comment by Daveyx0** [comment]
Thanks for the report. Should be fixed in the 1.2 alpha (finally)
